renv/profiles/dev/renv/settings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "bioconductor.version": null,
+  "external.libraries": [],
+  "ignored.packages": [],
+  "package.dependency.fields": [
+    "Imports",
+    "Depends",
+    "LinkingTo"
+  ],
+  "ppm.enabled": null,
+  "ppm.ignored.urls": [],
+  "r.version": [],
+  "snapshot.type": "explicit",
```

Comment:
You can set the snapshot type for the entire profile once it's loaded, such that `renv::snapshot()` for the `dev` profile will only ever capture the dependencies in the `DESCRIPTION` file, even without `type = "explicit"`.

Reply:
**[Suggestion, non-blocking]** This is great! Perhaps we could update the README steps for updating lockfiles to remove the suggestion to pass the `type` flag to `snapshot` calls?

https://github.com/ccao-data/model-res-avm/blob/fd873f45b4c5ecb05582dbac90a94dc17dbbeb3b/README.Rmd#L791

Reply:
Ehhh, even though it does it automatically now I feel like it's a good idea to signal that it differs from what is normally happening.